### DISCUSSION TITLE
Local archive not working correctly?

### DISCUSSION THREAD

**Discussion by cryptoisthefutur2000** [ericaltendorf/plotman#805] [Q&A] [answered]
I recently realized that i need to use the archive feature in order to not have my drives overflow. I have 32 drives connected locally to my computer through USB.  Plotman interactive makes it look like it doesn't know where to archive. Does this look right? According to "chia farm summary" I have 240 plots across all 32 drives. 

Various lines from plotman interactive:

<A>rchival: (active) No plots found ## I have 240 plots across all drives
archive=/mnt/farm/Chia (remote) ## should this say local?
Archive dirs free space
.:16809G | | | | | | | | | | | | | | | | | | ## I don't even know what's happening here. Should I see the drive names? I would think I would at least see 32 spaces?

My Config: ## this is copied from the documentation
archiving:
  target: local_rsync
  env:
    site_root: /mnt/farm/

All 32 drives are mounted under /mnt/farm/Chia Plots 0, /mnt/farm/Chia Plots 1..../mnt/farm/Chia Plots 31

As far as I can tell I've followed the archive instructions exactly for local archiving. Am I missing something? is the issue that I have spaces in the names of the drives?

**Comment by cryptoisthefutur2000** [comment, marked as answer]
The issue is the spaces in the names of the mounted drives. When I removed those spaces, it appears to be working better. I'll change all the names and report back.

**Reply by cryptoisthefutur2000** [reply]
I removed the spaces from all of my drive names. I now see all of the drives are listed under "Archive dirs free space" but I still have two questions:

    Why does it still say "No plots found"? there should be 240 plots across all those drives.
    Why does it say my archive is "(remote)"?

**Reply by altendky** [reply]
I believe that `No plots found` refers to dst not the archive targets?  I don't recall plotman counting plots on target drives.  I think the `(remote)` bit is just a bad hard coded label.  And yes, we need to fix handling of paths with spaces, sorry for that.

**Reply by cryptoisthefutur2000** [reply]
That makes sense. I think I had assumed it was the other because when I was using all the drives as dst it told me how many plots were on each.

And no worries on the rest, just good to know.  Thanks for the help!!
